CrossBrowserTestingTunnel.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * @module digdug/CrossBrowserTestingTunnel
+ */
+
+var fs = require('fs');
+var os = require('os');
+var pathUtil = require('path');
+var request = require('dojo/request');
+var Tunnel = require('./Tunnel');
+var util = require('./util');
+
+/**
+ * A CrossBrowserTesting tunnel.
+ *
+ * @constructor module:digdug/CrossBrowserTestingTunnel
+ * @extends module:digdug/Tunnel
+ */
+function CrossBrowserTestingTunnel() {
+	this.apikey = process.env.CBT_APIKEY;
+	this.username = process.env.CBT_USERNAME;
+	this.servers = [];
+	Tunnel.apply(this, arguments);
+}
+
+var _super = Tunnel.prototype;
+CrossBrowserTestingTunnel.prototype = util.mixin(Object.create(_super), /** @lends module:digdug/CrossBrowserTestingTunnel# */ {
+	constructor: CrossBrowserTestingTunnel,
+
+	/**
+	 * The CrossBrowserTesting API key.
+	 *
+	 * @type {string}
+	 * @default the value of the CBT_APIKEY environment variable
+	 */
+	apikey: null,
+
+	/**
+	 * The CrossBrowserTesting username.
+	 *
+	 * @type {string}
+	 * @default the value of the CBT_USERNAME environment variable
+	 */
+	username: null,
+
+	directory: pathUtil.join(__dirname, 'CrossBrowserTesting'),
+
+	executable: 'java',
```

Comment:
Is there any particular reason to use the Java version of the tunnel instead of the Node one?